### PR TITLE
Clock module

### PR DESCRIPTION
**Pull request by joanb**
Add module for clocks business logic.
Most of the work here has been to get jacoco working in a multimodule (android vs java) project with codecov. 
Clocks are still on it's way but will be iterated in another PR

### CODE REVIEW

File: clocks/build.gradle [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        xml.enabled = false
+        html.enabled = true
+    }
+    getSourceDirectories().setFrom(files(["src/main/java"]))
+    getClassDirectories().setFrom((file("$project.buildDir/intermediates/classes/debug").exists()
+        ? fileTree(dir: "$project.buildDir/intermediates/classes/debug")
+        : fileTree(dir: "$project.buildDir/classes")))
+    getExecutionData().setFrom(fileTree(dir: "$project.buildDir", includes: ["jacoco/*.exec"]))
+}
+*/
```
Comment (SergiLlamasGuerrero) [inline clocks/build.gradle]: Is this task necessary or could it be useful in the future? If not, the comment could be removed. What do you think? ✂️

Comment (joanb) [reply clocks/build.gradle]: This pull request is a draft 😅 I'm struggling with jacoco tasks to make it multimodule.
I made the draft to be able to launch travis and test the changes i'm introducing :)

Comment (SergiLlamasGuerrero) [reply clocks/build.gradle]: oops... I didn't pay attention to the draft state u.u'

Comment (joanb) [reply clocks/build.gradle]: @all-contributors please add @tbenning for review

Comment (joanb) [reply clocks/build.gradle]: @all-contributors please add @SergiLlamasGuerrero  for review

Comment (joanb) [reply clocks/build.gradle]: Deleted!

**Comment by joanb** [comment]
@all-contributors please add @SergiLlamasGuerrero for review

**Pull request merged**